plugins/woocommerce-admin/client/marketing/data-multichannel/resolvers.ts

```diff
@@ -47,3 +55,48 @@ export function* getRecommendedChannels() {
 		throw error;
 	}
 }
+
+const getTotalFromResponse = ( response: Response ) => {
+	return (
+		parseInt( response.headers.get( 'x-wp-total' ) || '0', 10 ) || undefined
+	);
+};
+
+export function* getCampaigns( page: number, perPage: number ) {
```

Comment:
💅  Missed JSDOCS

Reply:
Done in [`4b87f0a` (#36735)](https://github.com/woocommerce/woocommerce/pull/36735/commits/4b87f0a4ed13d00bd8054687bdab823ba10a6a03).